lib/elixir/lib/application.ex

```diff
@@ -145,6 +145,19 @@ defmodule Application do
   """
   @callback stop(state) :: term
 
+  @doc """
+  Called after an application starts.
```

Comment:
Could we improve this description? Btw this callback is called before the application has finished starting, that is before `Application.start` (etc) returns but after `start/2`.